v0/destinations/hs/HSTransform-v2.js

```diff
@@ -52,18 +71,43 @@ const processIdentify = async (message, destination, propertyMap) => {
   const traits = getFieldValueFromMessage(message, "traits");
   const mappedToDestination = get(message, MappedToDestinationKey);
   const operation = get(message, "context.hubspotOperation");
+  const externalIdObj = getDestinationExternalIDObject(message, "HS");
+  const { objectType } = getDestinationExternalIDInfoForRetl(message, "HS");
   // build response
   let endpoint;
   const response = defaultRequestConfig();
   response.method = defaultPostRequestConfig.requestMethod;
+
+  // Handle hubspot association events sent from retl source
+  if (
+    objectType &&
+    objectType.toLowerCase() === "association" &&
+    mappedToDestination
```

Comment:
`mappedToDestination && GENERIC_TRUE_VALUES.includes(mappedToDestination.toString())` - to handle true and "true" and other cases around it